tests/neural_compressor/test_neural_coder_adaptor.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+from optimum.intel.neural_compressor.neural_coder_adaptor import NeuralCoderAdaptor
+
+
+class NeuralCoderAdaptorTest(unittest.TestCase):
+    def test_string_type(self):
```

Comment:
Should we add a test to verify that model are correctly quantized using the launcher ?

Reply:
We can add a test on Launcher in some way. It's a bit hard to structure the UT for Launcher... Let me think about it!